package.json

```diff
@@ -28,25 +31,21 @@
   },
   "homepage": "https://github.com/slurmulon/gig#readme",
   "dependencies": {
-    "bach-js": "github:slurmulon/bach-js",
+    "@babel/runtime": "^7.14.0",
```

Comment:
Not sure if we still need this, need to test without (can still merge since it works, bundle optimization thing).